cpp/src/parquet/arrow/reader_writer_benchmark.cc

```diff
@@ -736,8 +737,9 @@ static void BM_ReadIndividualRowGroups(::benchmark::State& state) {
     auto reader =
         ParquetFileReader::Open(std::make_shared<::arrow::io::BufferReader>(buffer));
     std::unique_ptr<FileReader> arrow_reader;
-    EXIT_NOT_OK(FileReader::Make(::arrow::default_memory_pool(), std::move(reader),
-                                 &arrow_reader));
+    auto reader_result =
```

Comment:
Can we use `ASSERT_OK_AND_ASSIGN` here?

Reply:
No. `ASSERT_OK_AND_ASSIGN()` is for Google Test. Benchmarks uses Google Benchmark not Google Test.

Reply:
Thanks. Change to `EXIT_NOT_OK`.